src/components/App.js

```diff
@@ -5,10 +5,8 @@ import Display from './Display';
 function App() {
   return (
     <div className="App">
-      <div className="calc-wrapper">
-        <Display />
-        <ButtonPanel />
-      </div>
+      <Display />
+      <ButtonPanel />
     </div>
```

Comment:
Oops, good job trying to make changes.

Let's fix the React Fragment together.

check out this example:
```
    <>
      <Display />
      <ButtonPanel />
    </>
   ```

And if you really wanna style 

we could do this:
```
  <>
    <div>
      <Display />
      <ButtonPanel />
    </div>
  </>
  ```

Reply:
@bellom thank you for the review. Can you kindly check the code again. You comment is made rather on an old version

Reply:
@nganifaith Oops, let's double-check, I think I commented on your new changes, as you can see up there, line 8-11 was replaced.

Let's look into the file;
![image](https://user-images.githubusercontent.com/31897434/127193123-c3bd0d96-6d6a-4c6c-a8d8-6cd846cdc783.png)


@nganifaith I hope I was able to make everything more clear now.

Reply:
@Bellom Thank you for the review, however is an unnecessary change. Which problem are you trying to solve with the fragment? I am using the class for styling. Please read the best practices. 
This isn't a use case for a Fragment I am calling two different components which are wrapped in one container. I understood in the first review I had two unnecessary containers. Look at the code again and tell me why I need a fragment in this case. And Please feel free to show the other to others

Reply:
Apology, I think you can add style it is optional.

So if you stick to the style, we don't need the fragment as you said, sorry for the mixed up.